addons/account/models/account_payment.py

```diff
@@ -130,6 +130,9 @@ def _get_default_journal(self):
         compute='_compute_show_require_partner_bank',
         help="Technical field used to know whether the field `partner_bank_id` needs to be required or not in the payments form views")
     country_code = fields.Char(related='company_id.country_id.code')
+    amount_relative_to_type = fields.Monetary(
+        currency_field='currency_id', compute='_compute_amount_relative_to_type',
+        help='Negative value of amount field if payment_type is outbound')
```

Comment:
to stay consistent with invoice's fields, I'd rather name it amount_signed

Reply:
renamed